web/src/__tests__/ProfileEdit.test.tsx

```diff
@@ -123,11 +123,11 @@ function renderQuota() {
   const utils = render(
     <QuotaCardEdit
       profileId={profileIdProp}
-      quotaDetails={{
-        licensePlate: licensePlateProp,
-        quotaSize: quotaSizeProp,
-        quotaOptions: quotaOptionsProp,
-      }}
+      licensePlate={licensePlateProp}
```

Comment:
why is is licencePlateProp and not jsut licencePlate. It is a prop and is going in as a prop